stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -591,7 +605,8 @@
 (define-public (stack-stx (amount-ustx uint)
                           (pox-addr (tuple (version (buff 1)) (hashbytes (buff 32))))
                           (start-burn-ht uint)
-                          (lock-period uint))
+                          (lock-period uint)
+                          (signing-key (buff 33)))
```

Comment:
Can you add a comment about `signing-key` to the method doc?

Reply:
Updated a sentence & added the following to comment doc above function:

`;; * You need to provide a signing-key used to allocate signature-weight to a valid signer`